components/profile-controller/controllers/profile_controller.go

```diff
@@ -377,6 +379,14 @@ func (r *ProfileReconciler) updateIstioRbac(profileIns *profilev1.Profile) error
 			}
 		}
 	}
+
+	// Check all options here https://istio.io/docs/reference/config/policy-and-telemetry/attribute-vocabulary/
+	var propertyKey string
+	if r.AttributeType == "map" {
```

Comment:
Would it possible to write a unittest to ensure we generate the expected ServiceRoleBinding?

Reply:
Definitely. I should. I will update the PR later.